src/bin.ts

```diff
@@ -72,6 +72,12 @@ const cliDefinition = yargs(hideBin(process.argv)).command(
         type: 'boolean',
         describe: 'Generate React bindings (Provider and hook)',
       })
+      .option('experimental_proxyImpl', {
```

Comment:
is the plan to add it as an option, or eventually just replaced the approach from big `object` to the `proxy` approach?
I think it won't be breaking changes, and you should make it the only option.
can you think of any reason it will break for consumers?

Reply:
plan was to eventually replace to proxy as default
I played safe with the flag
but in general I'm also good with replacing it and having it enabled by default (bundlu reducing boost right away on new version!)

I will rework it and get back

Reply:
@varzager done ✅